ip_camera_ros2:
  ros__parameters:
    # SDK configuration
    image_topic: /image
    cam_info_topic: /camera_info
    image_height: -1
    image_width: -1
    offset_x: -1
    offset_y: -1
    url: rtsp://admin:password@ip
    tf: "camera_optical_link"
    frame_rate: 30
    # Camera calibration info
    enable_cam_info: true # true enables camera info publishing
    # Paste here your camera calibration parameters following the format:
    camera_matrix:
      [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    distortion_model: equidistant
    distortion_coefficients: [0.0, 0.0, 0.0, 0.0]
    rectification_matrix: [1., 0., 0., 0., 1., 0., 0., 0., 1.]
    projection_matrix:
      [
        1.0,
        0.0,
        1.0,
        0.0,
        1.0,
        0.0,
        1.0,
        0.0,
        1.0,
        0.0,
        1.0,
        0.0
      ]

